tox.ini

```diff
@@ -31,3 +32,12 @@ commands = sphinx-build -W -b html -d {envtmpdir}/doctress docs {envtmpdir}/html
 deps = pre-commit
 skip_install = true
 commands = pre-commit run --all-files --show-diff-on-failure
+
+
+[testenv:nightly]
+deps = 
```

Comment:
right now all this does is run the install and nothing else -- we should add a `commands` section and run the unit tests against this.

Reply:
no, it does run them since it inherits the command from the base `testenv`

Reply:
ah, right you are, I have actually never made a `testenv` where a command wasn't defined so I didn't realize this was the behavior.